src/Components/TeamDetails/TeamDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "./TeamDetails.module.css";
+import ONE from "../../assets/Team/one-team.png";
+import TWO from "../../assets/Team/two-team.png";
+import THREE from "../../assets/Team/three-team.png";
+const TeamDetails = props =>{
+    return (
+        <div className = {styles.main}>
+            <div className = {styles.one}>
```

Comment:
if all the child div has same css properties then give them same class, no need of 'one', 'two', 'three'
same for oneImg, twoImg and ThreeImg give same class name